.cm/gitstream.cm

```diff
@@ -0,0 +1,81 @@
+# -*- mode: yaml -*-
+manifest:
+  version: 1.0
+
+automations:
+  # Enforces a certain format on PR titles
+  enforce_pr_title:
+    if:
+      - {{ pr.title | match(regex=titlePolicy.titleRegex) | nope }}
+    run:
+      - action: request-changes@v1
+        args:
+          comment: |
+            All PRs must be titled according to our semantic naming policy: `[Area]: <short summary>`
+
+            Area must be one of the following:
+             * DataGrid
+             * Pickers 
+             * Charts
+             * TreeView
+             * Chore
+  # Add a label that indicates how many minutes it will take to review the PR.
+  estimated_time_to_review:
+    if:
+      - true
+    run:
+      - action: add-label@v1
+        args:
+          label: "{{ calc.etr }} min review"
+          color: {{ colors.error if (calc.etr >= 20) else (colors.warning if (calc.etr >= 5) else colors.success) }}
+  # Post a comment that lists the best experts for the files that were modified.
+  explain_code_experts:
+    if:
+      - { { pr.labels | match(term='Auto assign') | some } }
+    run:
+      - action: explain-code-experts@v1
+        args:
+          gt: 10
+  # Assign a random reviewer when the 'Share knowledge' label gets added
+  share_knowledge:
+    if:
+      - { { pr.labels | match(term='Share Knowledge') | some } }
+    run:
+      - action: add-reviewers@v1
+        args:
+          reviewers: { { repo | codeExperts(gt=30, lt=60) | random } }
+      - action: add-comment@v1
+        args:
+          comment: |
```

Comment:
If it costs $50 for every unique contributors / month https://linearb.io/pricing, is it worth ~$50k/year?

<img width="936" alt="Screenshot 2023-09-11 at 16 30 31" src="https://github.com/mui/mui-x/assets/3165635/f719e9ab-b62d-4545-8493-9c86a22702d6">